app/views/layouts/_footer.html.erb

```diff
@@ -44,7 +44,7 @@
           </div>
 
           <div class='col-sm-6 col-sm-offset-1'>
-            <div style="container">
+            <div class="container">
```

Comment:
Lets not include this file / change here – already worked on cleaning up the footer.